src/core/server/rendering/rendering_service.tsx

```diff
@@ -131,28 +122,21 @@ export class RenderingService {
               uiSettings: settings,
             },
             branding: {
+              darkMode,
```

Comment:
for consistency should this be 
darkMode: darkMode

Reply:
Yea, I originally did darkMode: darkMode, but yarn lint:es will complain about "expected property shorthand", and --fix flag will fix that into just "darkMode".